.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### General information
+
+  * Minishift version: 
+  * Download from: github releases / brew / chocolatey
+  * OS: Linux / macOS / Windows
+  * OS version/distro: 
+  * Hypervisor: KVM / xhyve / Hyper-V / VirtualBox
+  * `minishift start --shwo-libmachine-logs -v5`
+    If long, please consider posting this on http://gists.github.com/ and post the link in your issue
+
+### Problem description
```

Comment:
I think title will include part of description so this is unnecessary IMO too for issue template context.